ccaas_builder/cmd/detect/main.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var logger = log.New(os.Stderr, "", 0)
+
+func main() {
+	logger.Println("::Detect")
+
+	if err := run(); err != nil {
+		logger.Printf("::Error: %v\n", err)
+		os.Exit(1)
+	} else {
```

Comment:
No need for an else. You can handle the normal flow outside an indented else.
See guidance at https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow